openshift-knative-operator/cmd/openshift-knative-operator/kodata/knative-serving/latest/2-serving-core.yaml

```diff
@@ -9164,7 +9164,7 @@ webhooks:
     - key: app.kubernetes.io/component
       operator: In
       values: ["autoscaler", "controller", "logging", "networking", "observability", "tracing", "net-certmanager"]
-  timeoutSeconds: 10
+  timeoutSeconds: 30
```

Comment:
I thought we wanted to decrease it  (so that the delay is not that long if the issue occurs? )

Reply:
Ops my bad then, I have misinterpreted the meaning of this timeout.